### DISCUSSION TITLE
Multilanguage Content Support

### DISCUSSION THREAD

**Discussion by 27AR456** [retypeapp/retype#327] [General]
Dear Team, 

I have read your documentation and can't find any hint to multilanguage support for multilanguage content.

What we want is, to provide content in different languages. This means we have one file with content in different languages and the system will show the content regarding the browser language of the visitor. If the browser language is not the preferred language of the visitor the version should have the option to manually select the language on the site.

Should I move this to a Idear or let this stay her in General?

Thanks for Support.

**Comment by H8X24A1** [comment]
We would like to have this feature too. Please make this possible.
